zebra-rpc/src/methods.rs

```diff
@@ -29,6 +29,12 @@ use zebra_node_services::{mempool, BoxError};
 #[cfg(test)]
 mod tests;
 
+/// The RPC error code used by `zcashd` for missing blocks.
+///
+/// `lightwalletd` expects error code `-8` when a block is not found:
+/// https://github.com/adityapk00/lightwalletd/blob/c1bab818a683e4de69cd952317000f9bb2932274/common/common.go#L251-L254
```

Comment:
Renders nicer in the rustdoc:

```suggestion
/// <https://github.com/adityapk00/lightwalletd/blob/c1bab818a683e4de69cd952317000f9bb2932274/common/common.go#L251-L254>
```